src/processor/java/org/truffleruby/processor/BuildInformationProcessor.java

```diff
@@ -101,13 +105,15 @@ private void processBuildInformation(TypeElement element) throws IOException {
         }
     }
 
-    public String getRevision() {
-        return "f00fc3fc33";
+    public String getRevision() throws IOException {
+        final Process git = Runtime.getRuntime().exec("git rev-parse --short HEAD");
+        try (BufferedReader reader = new BufferedReader(new InputStreamReader(git.getInputStream()))) {
+            return reader.readLine();
+        }
     }
 
     public String getCompileDate() {
-        return "2018-01-07";
+        return new SimpleDateFormat("yyyy-MM-dd").format(new Date());
```

Comment:
I think it would be better to have the `commit date` of the latest commit.
In Ruby, this is exposed as `RUBY_RELEASE_DATE`, not a compile date.

Reply:
Fixed.